tools/functional-tester/test

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+set -x
+set -e
+
+# make sure it's linux64 
+export GOOS=linux
+export GOARCH=amd64
+
+# 1. build etcd binaries
+[ -f bin/etcd ] || ./build
```

Comment:
you're right - may be annoying - added explicit info for OSX users and removed this variables (it easy to spot an error, because scripts ends up with `format exec error`)